README.md

```diff
@@ -139,6 +139,7 @@ Hurray! You successfully made a contribution! 🎉
 | [D](#d) | [J](#j) | [P](#p) | [V](#v) |
 | [E](#e) | [K](#k) | [Q](#q) | [W](#w) |
 | [F](#f) | [L](#l) | [R](#r) | [X](#x) |
+
```

Comment:
Why is this line added?

Reply:
For some reason, VSCode adds the line when I save the README.MD file.

Reply:
```suggestion
```

Reply:
```suggestion
```

This should remove this.